src/qt/optionsmodel.cpp

```diff
@@ -28,6 +28,8 @@
 #include <QSettings>
 #include <QStringList>
 
+const char *DEFAULT_GUI_PROXY_HOST = "127.0.0.1";
```

Comment:
Why not put this in the same place as `DEFAULT_GUI_PROXY_PORT`?

Reply:
It is there, just not the value. 
Strings are pointers which point into a .rodata segment, so need to be present in a compilation unit.
Defining string constants in header files is a bad idea because it causes duplication in every place it's used, or even can lead to linker conflicts.